cmd/yggctl/main.go

```diff
@@ -131,6 +131,57 @@ func main() {
 			},
 			Action: dispatchAction,
 		},
+		{
+			Name:        "message-journal",
+			Usage:       "Display a list of all worker messages and emitted events",
+			UsageText:   "yggctl message-journal",
+			Description: "The message-journal command sends a dbus call to yggdrasil to retrieve a message journal containing all worker messages and emitted events.",
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name:     "persistent",
+					Aliases:  []string{"p"},
+					Usage:    "Display worker messages and emitted events gathered across multiple sessions from persistent storage.",
```

Comment:
Nit-pick: Usage strings don't end with periods. They don't need to be complete
sentences. They can be short, descriptive fragments.